makefile_lib

```diff
@@ -47,19 +47,19 @@ LST:=$(SRC_C:.c=.lst)
 INC:=-I$(INC_DIR) -I$(SRC_DIR) -I$(RES_DIR)
 
 # default flags
-DEF_FLAGS_M68K:=-m68000 -Wall -fno-builtin $(INC)
+DEF_FLAGS_M68K:=-m68000 -Wall -Wextra -Wno-shift-negative-value -fno-builtin $(INC)
 DEF_FLAGS_Z80:=-i$(SRC_DIR) -i$(INC_DIR)
 
 release: envcheck
-release: FLAGS:=$(DEF_FLAGS_M68K) -O3 -fno-web -fno-gcse -fno-unit-at-a-time -fomit-frame-pointer
+release: FLAGS:=$(DEF_FLAGS_M68K) -O3 -fuse-linker-plugin -fno-lto -fno-web -fno-gcse -fno-unit-at-a-time -fomit-frame-pointer -flto
```

Comment:
LTO is disabled so object files can be disassembled if needed. Is there a benefit to enabling it?
In any case, you have enabled it at the end with -flto, but did not remove -fno-lto. It should  be one or the other.

Reply:
In SGDK's makefile, LTO is enabled, so naturally I have it active here. However, you are right, it's not necessary to keep -fno-lto.